src/snowflake/cli/api/commands/snow_typer.py

```diff
@@ -111,3 +112,73 @@ def post_execute():
 
         log.debug("Executing command post execution callback")
         flush_telemetry()
+
+
+@dataclasses.dataclass
+class SnowTyperCommandData:
```

Comment:
Let's add comment what's the purpose of the class

Reply:
done!